app/_kong_plugins/ai-semantic-cache/changelog.md

```diff
@@ -3,4 +3,19 @@ content_type: reference
 no_version: true
 ---
 
-## Changelog
\ No newline at end of file
+## Changelog
+
+### {{site.base_gateway}} 3.9.0.0
+* Added the `ignore_tool` configuration option to discard tool role prompts from the input text.
+* This plugin can now be enabled on consumer groups.
+* Made the `embeddings.model.name` config field a free text entry, enabling use of a self-hosted (or otherwise compatible) model.
```

Comment:
```suggestion
* Made the `config.embeddings.model.name` config field a free text entry, enabling use of a self-hosted (or otherwise compatible) model.
```